scripts/install.js

```diff
@@ -357,7 +357,7 @@ function provideCompiler() {
     var releaseNinja = require("./ninjaFactory.js").libNinja({
       ocamlopt: "ocamlopt.opt",
       ext: ".exe",
-      INCL: require("./buildocaml.js").getVersionPrefix()
+      INCL: ocamlVersion
     });
```

Comment:
where does this variable come from in this file?

Reply:
This comes from line 30.
https://github.com/BuckleScript/bucklescript/pull/3655/files/5d3a3cb66c7d0433a6edc9caf67b59f99dbc0ad1#diff-153f7a81745c9180edcfa4838beeae7bL360